pkg-manifest/exportable-manifest/src/index.ts

```diff
@@ -63,7 +65,12 @@ export async function createExportableManifest (
     publishManifest.readme ??= opts.readmeFile
   }
 
-  return publishManifest
+  const config = await getConfig({}, {
+    excludeReporter: false,
+    rcOptionsTypes: {},
+    workspaceDir: await findWorkspaceDir(process.cwd()),
+  })
+  return (await config.hooks?.readPackageForPublishing?.[0]?.(publishManifest)) ?? publishManifest
```

Comment:
This allows the hook to mutate the value of `publishManifest` and not return anything, or return a new object.